package.json

```
@@ -25,7 +25,8 @@
   },
   "scripts": {
     "start": "npm run dev",
-    "dev": "nodemon --watch ./src"
+    "dev": "nodemon --watch ./src",
+    "test": "node --inspect node_modules/mocha/bin/_mocha --watch --no-timeouts"
```

Test command should check for linting errors as well. Additionally, there should be two seperate commands for watch mode and non-watch mode.
```suggestion
    "lint": "eslint \"./*.js\" \"./**/*.js\"",
    "test": "npm run lint --silent && ./node_modules/mocha/bin/_mocha",
    "test:watch": "nodemon --ext .js -w ./src/ -w ./test/ --exec \"npm run test --silent || true\"",
```